CONTRIBUTING.md

```diff
@@ -76,11 +76,25 @@ const mockCoinDetails = {
 };
 ```
 
-- For example, if you go to route localhost:8000/coins/alex and you have an input called "authToken" inside the headers set to "123" then you could receive his balance of 100 tokens. If the username or authToken is not set correctly inside the headers, then this route will throw an error  explaining that they are not permitted to access this route. 
+- For example, if you go to route localhost:8000/coins/alex and you have an input called "authtoken" inside the headers set to "123" then you could receive his balance of 100 tokens. If the username or authToken is not set correctly inside the headers, then this route will throw an error  explaining that they are not permitted to access this route.
+
+## Testing 
+
+A testing folder is in place that will test all three of these basic API routes. Before creating your PR please make sure all of the unit tests pass.
+
+To run these unit tests, run your local server at localhost8000
+
+```bash
+cd tests
+npm install
+npm run test
+```
+- Please note that even if you don't have the server running, at least 1 unit test should pass ensuring that you have the tests folder configured/installed properly.
+- For more testing requirements, please check the tests [README.md](https://github.com/promethean-tech/rest-api-comparison/tree/dev/tests)
 
 ## Documentation
```

Comment:
If you also want to style this documentation with emojis, headers, etc, chatGPT is really good for this. Most of the time I paste in all the content into there and ask it to format it for me nicely

Reply:
Good Idea, will do that now to see if it looks nicer